modules/rtm/R/invert.custom.R

```diff
@@ -242,32 +262,41 @@ invert.custom <- function(observed, invert.options,
     # Metropolis sampling step if all conditions have been met
```

Comment:
I implemented these differently to allow the use of less formal priors. For instance, `dnorm` is technically never not finite, but it might be convenient to assign a tight `dnorm` prior on a value.

Regarding your second question, I reject the parameters to force the Jump distribution to shrink. That way, if the sampler keeps proposing illegal values, the acceptance rate is low, and the jump distribution will narrow until it gets back to drawing mostly legal values.